src/main/java/org/dependencytrack/event/ApplyRetentionPolicyEvent.java

```diff
@@ -0,0 +1,10 @@
+package org.dependencytrack.event;
+
+import alpine.event.framework.Event;
+
+/**
+ * Defines an event used to apply the general retention policy
+ */
+public class ApplyRetentionPolicyEvent implements Event {
```

Comment:
Knowing that a BOM retention policy is coming, I think this class name should be changed to `ApplyProjectRetentionPolicyEvent` so that its clear it is project-related.

Reply:
Hi @stevespringett , thank you for the review. It makes sense, I updated the name everywhere.